apis/proto/v1/payload/payload.proto

```diff
@@ -116,6 +116,8 @@ message Search {
     string request_id = 1;
     // Search results.
     repeated Object.Distance results = 2;
+    // The value is related to the request key.
+    optional Meta.Value value = 3;
```

Comment:
What type of search is supported?
For example, are `key = value (vectors with the key-value pair)` or `key (vectors with the key)` supported?